styx-flyte-client/src/test/java/com/spotify/styx/flyte/client/FlyteAdminClientTest.java

```diff
@@ -36,10 +36,10 @@
 
 public class FlyteAdminClientTest {
 
-  private static final String DOMAIN = "testing";
-  private static final String PROJECT = "styx_flyte_test";
-  private static final String LP_NAME = "launch_plan_1";
-  private static final String LP_VERSION = "launch_plan_version_1";
+  protected static final String PROJECT = "styx_flyte_test";
+  protected static final String DOMAIN = "testing";
+  protected static final String LP_NAME = "launch_plan_1";
+  protected static final String LP_VERSION = "launch_plan_version_1";
```

Comment:
```suggestion
  static final String PROJECT = "styx_flyte_test";
  static final String DOMAIN = "testing";
  static final String LP_NAME = "launch_plan_1";
  static final String LP_VERSION = "launch_plan_version_1";
```
This could be just package as we don't expect classes to extend from `FlyteAdminClientTest`